scripts/build/builders/qpg.py

```diff
@@ -27,7 +27,7 @@ def __init__(self, root, runner):
     def build_outputs(self):
         return {
             'chip-qpg-lock-example.out':
-                os.path.join(self.output_dir, 'chip-qpg6100-lock-example.out'),
+                os.path.join(self.output_dir, 'chip-qpg6105-lock-example.out'),
             'chip-qpg-lock-example.out.map':
                 os.path.join(self.output_dir,
                              'chip-qpg6100-lock-example.out.map'),
```

Comment:
please change map as well. These will be used to figure out how to extract the 'flash bundle' files during builds.

Reply:
Are there more qpg.py files ? I found this one and already changed.